_posts/2013-05-15-understanding-vestibular-disorders.md

```diff
@@ -31,4 +31,28 @@ The cause may be from illness, injury, or a genetic condition, but anyone can su
 
 ## What should you consider?
 
-Don't make animations, sliders, or rapid movement start automatically. Give an indicator of what movement will happen on the site when someone takes action. Allow the option to turn off any animation and movement at any point in the process.
+Don't make animations, sliders, videos, or rapid movement start automatically. Give an indicator of what movement will happen on the site when someone takes action. Allow the option to turn off any animation and movement at any point in the process.
+
+Also, with the CSS [`prefers-reduced-motion`](https://developer.mozilla.org/en-US/docs/Web/CSS/@media/prefers-reduced-motion) media query you can write conditional CSS animations and transitions based on the user's preference. For example, you may want to enable [smooth scrolling](https://developer.mozilla.org/en-US/docs/Web/CSS/scroll-behavior), but disable it and all other potential animations and transitions for users who explicitly prefers reduced motion:
```

Comment:
I like this addition a ton, but could we use an example other than smooth scrolling? 

It's not a very well-loved feature accessibility-wise, so I'd like to have the article not even suggest the hint that it's a property to use.

Reply:
@ericwbailey 

I've included both a transition and an animation, and removed `scroll-behavior`. I also clarified that it is in the browser settings that the preference is exposed from.

However, I don't think it's the right approach to not expose developers to certain CSS properties because they may inherently be considered worse in terms of accessibility than the absence of that property. I'd rather create awareness around the potential accessibility issue(s) of that property/value pair and show how to (in this case) use a preference-based media query to restore the default and least annoying behavior.

But the point of the update to this article is to at least mention `prefers-reduced-motion`, which we're doing here :)